### PR TITLE
Mobile done

### PR DESCRIPTION
**Pull request by amonamoqadi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,39 +9,109 @@
...
-                <img src="img/renovated.jpg" alt="renovated house" />
-                <p>Renovated House</p>
+    <!-- body section -->
+    <div class="body-sec">
+        <!-- Projects section -->
+        <div class="projects-sec">
+            <span>Projects</span>         
+            <div class="project-inner">
+                <div class="project-section-img"> 
+                    <img src="img/summer.jpg" alt="summer house" />
```
Comment (IbrahimLama) [inline index.html:32]: move property alt to a new line under src,
apply this to tags that has multi properties

File: index.html [hunk 2]
```diff
@@ -9,39 +9,109 @@
...
+                    <p class="career"> Architecter</p>                     
+                    <p class="parag">In addition to resize text and images, it is also common to use media queries in responsive web pages.</p>
+                    <button type="button"> Contact</button> 
+                </div>
+                <div class="about-img"> 
+                        <img src="img/dan.jpg" alt="mike" />  
+                        <p>Mike Ross </p>                        
+                        <p class="career"> Architecter</p>                    
+                        <p class="parag">In addition to resize text and images, it is also common to use media queries in responsive web pages.</p>
+                        <button type="button"> Contact</button>
```
Comment (IbrahimLama) [inline index.html:94]: not equal indentation

File: index.html [hunk 3]
```diff
@@ -9,39 +9,109 @@
...
+                        <p class="career"> Architecter</p>                    
+                        <p class="parag">In addition to resize text and images, it is also common to use media queries in responsive web pages.</p>
+                        <button type="button"> Contact</button> 
+                </div>
+                <div class="about-img"> 
+                        <img src="img/dan.jpg" alt="Dan" />                      
+                        <p>Dan Star </p>                      
+                        <p class="career"> Architecter</p>                     
+                        <p class="parag">In addition to resize text and images, it is also common to use media queries in responsive web pages.</p>
+                        <button type="button"> Contact</button>
```
Comment (IbrahimLama) [inline index.html:101]: fix the indentation

File: style.css [hunk 4]
```diff
@@ -1,109 +1,227 @@
-  *{
-   margin: 0;
-   height: 100%;
-  }
   body{
    width: 100%;
+   margin-left: 0px;
    font-family:  "Open Sans", Helvetica, sans-serif;
   }
   .img-container{
```
Comment (IbrahimLama) [inline style.css:6]: add line above, and apply this to all the selectors

File: style.css [hunk 5]
```diff
@@ -1,109 +1,227 @@
...
+  }
+  .top-panner{
+    position: fixed;
+    width: 100%;
+    top: 0;
+    padding: 10px;
+    z-index: 99;
+    background-color:white;
+    box-shadow:0 2px 5px 0 rgba(0,0,0,0.16),0 2px 10px 0 rgba(0,0,0,0.12);
+    font-family:  "Open Sans", Helvetica, sans-serif;
```
Comment (IbrahimLama) [inline style.css:24]: remove font-family as you are giving it to the body

File: style.css [hunk 6]
```diff
@@ -1,109 +1,227 @@
-  *{
-   margin: 0;
-   height: 100%;
-  }
   body{
    width: 100%;
+   margin-left: 0px;
```
Comment (IbrahimLama) [inline style.css:3]: why do we need margin-left: 0 here ?

File: style.css [hunk 7]
```diff
@@ -1,109 +1,227 @@
...
+    width: 100%;
+    top: 0;
+    padding: 10px;
+    z-index: 99;
+    background-color:white;
+    box-shadow:0 2px 5px 0 rgba(0,0,0,0.16),0 2px 10px 0 rgba(0,0,0,0.12);
+    font-family:  "Open Sans", Helvetica, sans-serif;
+    font-size: 15px;
+  }
+  .top-panner span{
```
Comment (IbrahimLama) [inline style.css:27]: add a space after span, and apply this to other selectors

File: style.css [hunk 8]
```diff
@@ -1,109 +1,227 @@
...
     transform: translate(-50%, -50%);
-    text-align: center;
+    margin-top:25px;
+    color: white;
     font-family: "Open Sans", Helvetica, monospace;
   } 
-  .inner-container b{ 
-    padding: 4% 4% 3% 4%;
+  .inner-container span{ 
+    padding: 15px 17px  10px;
```
Comment (IbrahimLama) [inline style.css:45]: there an extra space before 10px

File: style.css [hunk 9]
```diff
@@ -1,109 +1,227 @@
...
-    color:white;
-    top: 0px; 
+    color: rgb(235, 229, 229);
+    top: 0px;
     left: 0px;
-    text-align: center;
-    padding: 7px 10px 5px;
-    height: 17px;
-    font-size: small;
+    padding: 10px 15px 10px;
```
Comment (IbrahimLama) [inline style.css:79]: move top, left and padding under position

File: style.css [hunk 10]
```diff
@@ -1,109 +1,227 @@
...
+    padding: 10px;
+    border: none;
+    background-color: #000;
+    color: #fff;
+  }
+  .contact-sec input{
+    width: 100%;
+    height: 30px;
+    margin: 0px 0px 15px;
+    border:1px solid #ccc;
```
Comment (IbrahimLama) [inline style.css:127]: add a space after the :
apply this to all properties

File: style.css [hunk 11]
```diff
@@ -1,109 +1,227 @@
...
+    /* about section */
+    .about-sec:after{
+      content: "";
+      display: table;
+      clear: both;  
+    }
+    .about-img{
+      float: left;   
+      width: 25%; 
+    }
```
Comment (IbrahimLama) [inline style.css:225]: as you are not changing this from min-width: 650px, no need to re-write it

File: style.css [hunk 12]
```diff
@@ -1,109 +1,227 @@
...
+    width: 50%;  
+    padding-right: 10px;
+  }
+
 }
 @media only screen and (min-width: 1000px)
 {
- 
+  .img-container{
+    height: 550px;
```
Comment (IbrahimLama) [inline style.css:205]: reduce it to 400px

File: style.css [hunk 13]
```diff
@@ -41,187 +46,203 @@
...
     letter-spacing:3px;
   }
 
   /* body-sec */
-  .body-sec{
+  .body-sec {
     margin-right: 50px;
     margin-left: 50px; 
     margin-top: 70px;
     margin-bottom: 40px;
```
Comment (IbrahimLama) [inline style.css]: use margin shorthand

Comment (amonamoqadi) [reply style.css]: yes yes I did it

**Pull request merged**